website/dbt-versions.js

```diff
@@ -26,5 +26,9 @@ exports.versionedPages = [
   {
     "page": "docs/guides/migration-guide/upgrading-to-v1.1",
     "firstVersion": "1.1",
+  },
+  {
+    "page": "reference/dbt-jinja-functions/print",
```

Comment:
@JKarlavige This is not working. What have I done wrong?

Reply:
Hmm, it looks like it's working for me https://deploy-preview-1320--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/print

The banner shows at the top and I don't see it in the sidebar. Let me know if it's still not working on your end, happy to jump in a screen share.

Reply:
I see it now. I think I might have something messed up locally because I also see two `1.1` in  the dropdown list :/

Reply:
No worries! Easy for things to get out of sync with how many updates are added to this site. Let me know if you wanted to look into it further if it's still acting up